Geometry/TrackerNumberingBuilder/src/utils.cc

```diff
@@ -15,3 +16,18 @@ std::vector<uint32_t> TrackerGeometryUtils::getSiStripDetIds(const GeometricDet&
       std::begin(stripDetIds), std::end(stripDetIds), [](DetId a, DetId b) { return a.subdetId() < b.subdetId(); });
   return stripDetIds;
 }
+
+std::vector<uint32_t> TrackerGeometryUtils::getOuterTrackerDetIds(const GeometricDet& geomDet) {
+  std::vector<const GeometricDet*> deepComp;
+  geomDet.deepComponents(deepComp);
+  std::vector<uint32_t> OTDetIds;
+  for (const auto* dep : deepComp) {
+    const auto detId = dep->geographicalId();
+    if ((detId.det() == DetId::Detector::Tracker) && (detId.subdetId() > PixelSubdetector::PixelEndcap)) {
+      OTDetIds.push_back(detId);
```

Comment:
```suggestion
      OTDetIds.emplace_back(detId);
```

Reply:
`emplace_back` is preferred to `push_back`. I think there is even a Clang-tidy campaign that will make that conversion soon.

Reply:
mmh... even for a simple native C++ type as `uint32_t`? I think the two variants are totally equivalent in such a case.

Reply:
Indeed, I don't think it matters for primitive types.

Reply:
It's OK in this case, so no change is necessary, but I think the main idea is there is almost never a reason to use `push_back`. `emplace_back` is either equivalent or superior, so it might as well always be used.